azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,50 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge devices"] = """
+        type: group
+        short-summary: Commands to create and manage IoT Edge devices.
+    """
+
+    helps["iot edge devices create"] = """
+        type: command
+        short-summary: Create configuratations of edge devices in an IoT Hub.
+        long-summary: |
+          This operation accepts inline device arguments or an edge devices configuration file in YAML or JSON format.
+          Inline command args (like '--device-auth') will take precedence and override configuration file properties if they are provided.
+          A sample configuration file can be found here: https://github.com/Azure/azure-iot-cli-extension/tree/dev/docs/samples/sample_devices_config.yaml
+          Review examples and parameter descriptions for details on how to fully utilize this operation.
+
+        examples:
+        - name: Create a couple of edge devices using symmetric key auth (default)
+          text: |
+            az iot edge devices create -n {hub_name} --device id=device_1 --device id=device_2
+
+        - name: Create a flat list of edge devices using self-signed certificate authentication with various edge property configurations, using inline arguments.
+          text: |
+            az iot edge devices create -n {hub_name} --device-auth x509_thumbprint --default-edge-agent {default_edge_agent}
+            --device id=device_1 hostname={FQDN}
+            --device id=device_2 edge_agent={custom_agent}
+            --device id=parent hostname={FQDN} edge_agent={custom_agent} container_auth={path_or_json_string}
+
+        - name: Delete all existing devices on a hub and create new devices based on a configuration file (with progress bars and visualization).
+          text: >
+            az iot edge devices create -n {hub_name} --cfg path/to/config_yml_or_json -c -v
+
+        - name: Create a group of nested edge devices with custom module deployments - containing 2 parent devices with 1 child device each, using inline arguments.
+            Also specifies output path for device certificate bundles.
+          text: |
+            az iot edge devices create -n {hub_name} --out {device_bundle_path}
+            --device id=parent_1 deployment=/path/to/parentDeployment_1.json
```

Comment:
we may need a tsg for a person trying to add too many devices and hitting the command line input limit

Reply:
actually the shell will prob handle it and it should be pretty obvious if we get an issue about this. This can be ignored then